frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -138,28 +138,12 @@ export const NavBar: React.SFC<NavBarProps> = ({pages, basePath, hideDivider}) =
     {primaryTabs}
     {secondaryTabs}
   </div>;
-};
+});
 NavBar.displayName = 'NavBar';
 
-export class HorizontalNav extends React.PureComponent<HorizontalNavProps> {
```

Comment:
Converted to functional component with `useMemo()` to prevent unmounting child `<Route>` components.